tools/RAiDER/downloadGNSSDelays.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python3
```

Comment:
@sssangha can you also run the following command on each of the modified files to take care of whitespaces consistent with PR#58:
autopep8 --select E231,E225,E251,E111,E128,E127,E265,E303,E261,E301 --in-place --recursive  <filename>